src/less/variables-payex.less

```diff
@@ -157,7 +157,7 @@ body {
 	/* TODO: to be converted: --brand-primary-faded:  fade(var(--brand-primary, #2A9D3F), 60%); */
 
 	--brand-secondary: #3c3c3c;
-	--brand-secondary-hover: #5f5f5f;
+	--brand-secondary-hover: #707070;
```

Comment:
remember to keep "LESS vars" and "Pure CSS vars, i.e. custom properties" in sync 😉